File: MovieAppMVVMRetrofitJava/app/src/main/java/com/masterclass/movieappmvvmretrofitjava/api/MovieApiService.java
package com.masterclass.movieappmvvmretrofitjava.api;

import com.masterclass.movieappmvvmretrofitjava.models.Result;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface MovieApiService {
    @GET("movie/popular")
    Call<Result> getPopularMovies(
      @Query("api_key") String api_key
    );
}

File: MovieAppMVVMRetrofitKotlin/app/src/main/java/com/masterclass/movieappmvvmretrofitkotlin/api/MovieApiService.kt
package com.masterclass.movieappmvvmretrofitkotlin.api

import com.masterclass.movieappmvvmretrofitkotlin.models.MoviesResult
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface MovieApiService {
    @GET("movie/popular")
    fun getPopularMovies(
        @Query("api_key") api_key: String
    ): Call<MoviesResult>
}